image/api_selector.go

```diff
@@ -230,7 +236,15 @@ func (as *APISelector) buildCandidateTags(params *Params) []*tagSet {
 		sort.Strings(ts.Tags)
 	}
 
-	return result
+	for _, ts := range result {
+		for _, tag := range ts.Tags {
+			if strings.Contains(tag, ",") {
+				return result, workerErrors.NewWrappedJobAbortError(errors.Errorf("tag %v contained \",\", which is not supported by job-board -- check .travis.yml for trailing comma", tag))
```

Comment:
I definitely agree that this error should be caught at the gatekeeper level. But I do want to also handle it in worker as a fallback. If you think that is overkill/redundant, I'd be glad to hear your thoughts.

I'll see if I can make the error better, and potentially split it into a user facing one and a log-facing one.